backend/src/sdk.cpp

```diff
@@ -209,6 +240,147 @@ restinio::request_handling_status_t SDK::handleTxSDKCall(const restinio::request
     return req->create_response().set_body(absl::StrJoin(outData, ",")).done();
 }
 
+void SDK::handleSetStationState(const nlohmann::json json)
```

Comment:
So many lines of code. This does work but I am definitely open to suggestions for how to reduce the code duplication. I tried a few different ways and failed on all of them since my C++ isn't the best. Pretty much every if statement is doing the exact same thing, just with different json values and get/set methods. There's gotta be a less verbose way.

Reply:
I've got a better way to do this, will update PR in a few.